### PR TITLE
ImpCEvalFun

### PR DESCRIPTION
**Pull request by clayrat**

### CODE REVIEW

**Comment by yurrriq** [comment]
Hey, thanks! I'll look this over when I get a chance, likely tomorrow night.

**Comment by clayrat** [comment]
I've went through most of the exercises until the end of `ImpCEvalFun` and added some minor fixes/updates based on that. I'm still thinking about how to do `Extract` and `Auto` and then we're done with volume 1. We should probably also update the PDF, as I've heard from several people they were confused by the difference between it and the repo code.

Also, before we start on volume 2 I'm guessing we should make a folder `VolumeOne` or something, and move everything in there?

**Comment by yurrriq** [comment]
I'm having trouble getting stuff to build locally, but I'm working on a (Nix-based) toolchain overhaul. Don't let me get in your way. My free time is nearly nonexistent 😕

File: src/Imp.lidr [hunk 1]
```diff
@@ -601,7 +601,7 @@ together with a declaration of what the notation means.
 >   E_APlus : e1 \\ n1 -> e2 \\ n2 -> n = n1 + n2 ->
 >             (APlus0 e1 e2) \\ n
 
-\todo[inline]{We don't use \idr{-} since it requires a proof of \idr{LTE n1 n2}}
+\ \todo[inline]{We don't use \idr{-} since it requires a proof of \idr{LTE n1 n2}}
```
Comment (yurrriq) [inline src/Imp.lidr:604]: The various added `\ `&nbsp; like this after a code block and before ` \todo` broke the PDF build for me.

**Pull request closed without merging**